scripts/aws/bootstrap.py

```diff
@@ -49,48 +102,96 @@ def bootstrap_api(client, stage_name):
 
     return response['id']
 
-def create_base_path_mapping(client, api_id, api_base, stage_name):
+
+def create_base_path_mapping(api_id, api_base, stage_name):
     """Link a custom domain with an API Gateway object stage"""
 
+    client =  boto3.client('apigateway', region_name=args.aws_region)
+
     # Note: this will fail if a mapping already exists (bootstrapping is not idempotent).
     response = client.create_base_path_mapping(
         domainName=api_base,
         basePath='(none)',
         restApiId=api_id,
         stage=stage_name
-        )
+    )
     logging.info("Domain '%s' now pointed to '%s':'%s'", api_base, api_id, stage_name)
     return response
 
-def main(log_level=20):
-    """Update an AWS API Gateway object from a Swagger configuration file."""
 
-    logging.basicConfig(level=log_level, format='[%(asctime)s] %(levelname)s %(message)s')
+def file_arg_to_string(filename):
+
+    with open(filename, "r") as fin:
+        body = fin.read()
+
+    return body
+
+
+if __name__ == '__main__':
+
+    logging.basicConfig(level=20, format='[%(asctime)s] %(levelname)s %(message)s')
```

Comment:
Use the named level instead of the integer value (e.g. `logging.INFO`). I didn't realize they were integers. #TIL.